### PR TITLE
Canonical consent banner tests

### PR DESCRIPTION
**Pull request by jamesbrumpton**
Resolves #1413 

**Overall change:** _Adding Cypress tests for the Cookie Banner on canonical pages ._

**Code changes:**

- _Testing the banner appears._
- _Testing that the banner disappears once relevant buttons have been clicked._
- _Testing that the cookies are assigned the relevant values._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
   "video": false,
+   "chromeWebSecurity": false,
   "env": {
     "APP_ENV": "local"
+
```
Comment (dr3) [inline cypress.json:6]: Unneeded added line

File: cypress/integration/cookieBanner.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  beforeEach(() => {
+    cy.visit(`/news/articles/${config.assets.newsThreeSubheadlines}`);
+  });
+
+  it('should have a privacy & cookie banner, which disappears once "accepted" ', () => {
+    getPrivacyBanner().should('be.visible');
+    getCookieBanner().should('not.be.visible');
+
+    retrieveCookieValue('ckns_privacy', '1');
+    retrieveCookieValue('ckns_policy', '000');
```
Comment (dr3) [inline cypress/integration/cookieBanner.js:23]: We should call `expectCookieExpiryDateOneYear` on `ckns_privacy` and `ckns_policy` before banner interation

**Review by dr3 — CHANGES_REQUESTED** [review]
Can you please look into cypress network request testing https://docs.cypress.io/guides/guides/network-requests.html#Testing-Strategies

Would be good to test the request to `https://cookie-oven.api.bbc.co.uk/ckns_policy/*`

**Review by dr3 — CHANGES_REQUESTED** [review]
More info on the client side cookie oven request we should test for
```
Cypress on LIVE
Client side request is made to 'https://cookie-oven.api.bbc.com/ckns_policy/000'
Client side request is made to 'https://cookie-oven.api.bbc.com/ckns_policy/111'

Cypress on TEST
Client side request is made to 'https://cookie-oven.test.api.bbc.com/ckns_policy/000'
Client side request is made to 'https://cookie-oven.test.api.bbc.com/ckns_policy/111'

Cypress on LOCAL
Client side request is made to 'http://localhost:7080/ckns_policy/000'
Client side request is made to 'http://localhost:7080/ckns_policy/111'
```

**Comment by ChrisBAshton** [comment]
Closed in favour of https://github.com/bbc/simorgh/pull/1439

**Pull request closed without merging**